.env

```diff
@@ -104,3 +104,20 @@ REDIS_ENABLED=true
 REDIS_PORT=6379
 REDIS_HOST=localhost
 REDIS_PASSWORD=vivid
+
+# oauth2-proxy
+OAUTH2_PROXY_OIDC_ISSUER_URL=$IDP_SSO_URL
+OAUTH2_PROXY_CLIENT_ID=$IDP_CLIENT_ID
+OAUTH2_PROXY_CLIENT_SECRET=$IDP_CLIENT_SECRET
+OAUTH2_PROXY_UPSTREAMS=${ADMIN_URL_INTERNAL},${API_URL_INTERNAL}/
+OAUTH2_PROXY_REDIRECT_URL=${ADMIN_URL}/oauth2/callback
+OAUTH2_PROXY_HTTP_ADDRESS=0.0.0.0:${AUTHPROXY_PORT}
```

Comment:
isn't running oauth 2 proxy with a binary with`0.0.0.0` a problem, because when running not in a docker container, makes this tentitive available in the whole network?

Reply:
Absolutely. Changed with 16d97f1fcd25fa6506ec683e86c5e42f825b44d5